.github/workflows/pnp-build-autopilot-processor.yml

```diff
@@ -0,0 +1,252 @@
+name: Merge to master 
+
+on:
+  workflow_call:
+    secrets:
+      GCR_image_name:
+        description: 'A name of the image to be created in GCR'
+        required: true
+      path-to-pom:
+        description: 'Path to the pom file'
+        required: true
+      path-to-dockerfile:
+        description: 'Path to dockerfile which builds the service.'
+        required: true
+      slack-channel:
+        description: 'A slack channel which needs to be notified in case of failure'
+        required: false
+      SECRET_AUTH:
+        required: true
+      GCLOUD_AUTH_STAGING:
+        required: true
+    inputs:
+      service-definition:
+        description: 'Path to kubernetes config file.'
+        type: string
+        required: true
+      release-version:
+        description: 'deploy release version'
+        type: boolean
+        required: false
+        default: false
+      jdk-distribution:
+        description: 'JDK Distribution name'
+        type: string
+        required: false
+        default: 'temurin'
+      java-version:
+        description: 'Java version used in the project'
+        type: string
+        required: false
+        default: '17'
+      use-old-setup-java:
+        description: 'If true uses the java setup step of version 1'
+        type: boolean
+        required: false
+        default: false
+      reuse-release:
+        description: 'If true, instead of creating new release reuses the current one. Intended to be used in config-based multi-service deploy scenarios, when multiple service instances created from the same code base.'
+        type: boolean
+        required: false
+        default: false
+      tag-prefix:
+        description: 'Release tag prefix. Must be provided in case of multiple projects in the same repository. Each project needs to have its own prefix. Use {service-prefix}-v- format.'
+        type: string
+        required: false
+        default: 'v'
```

Comment:
description needs to be updated, in one discussion https://stefanini-extendareta.slack.com/archives/CKN704GBS/p1695644654855079 was decided to use 'v' instead o 'v-' to have release name processor-name-v3.0.9 instead of processor-name-v-3.0.9